dynawo/sources/Models/CPP/ModelNetwork/test/TestLoad.cpp

```diff
@@ -568,16 +571,14 @@ TEST(ModelsModelNetwork, ModelNetworkLoadJt) {
   load->evalJt(smj, 1., 0);
   smj.changeCol();
   smj.changeCol();
-  ASSERT_EQ(smj.nbElem(), 6);
-  ASSERT_DOUBLE_EQUALS_DYNAWO(smj.Ax_[0], 22.155644370746373);
-  ASSERT_DOUBLE_EQUALS_DYNAWO(smj.Ax_[1], -153.29392144688848);
-  ASSERT_DOUBLE_EQUALS_DYNAWO(smj.Ax_[2], -87.596526541079143);
-  ASSERT_DOUBLE_EQUALS_DYNAWO(smj.Ax_[3],  8.4895038000592429);
-  ASSERT_DOUBLE_EQUALS_DYNAWO(smj.Ax_[4],  2.3388791341676249);
-  ASSERT_DOUBLE_EQUALS_DYNAWO(smj.Ax_[5],  1.3365023623815);
+  ASSERT_EQ(smj.nbElem(), 4);
+  ASSERT_DOUBLE_EQUALS_DYNAWO(smj.Ax_[0], 2);
+  ASSERT_DOUBLE_EQUALS_DYNAWO(smj.Ax_[1], 8.4895038000592429);
+  ASSERT_DOUBLE_EQUALS_DYNAWO(smj.Ax_[2], 5.2397892818982124);
+  ASSERT_DOUBLE_EQUALS_DYNAWO(smj.Ax_[3], 2.9941653039418354);
```

Comment:
Aren't we missing 2 indexes here?

Reply:
No, the error in the test make a threshold be not passed so we have less 2 terms

Reply:
Are we sure about that ? seems to me that it was to test with the correct inputs

Reply:
OK